tests/common/config.sh

```diff
@@ -2,14 +2,16 @@
 
 set -e
 
-build_path=$(pwd)
-while [ "${build_path}" != "/" -a ! -e "${build_path}/.git" ]; do
-	build_path=$(dirname ${build_path})
-done
-if [ ! -e "${build_path}/.git" ]; then
-	echo "Not inside git directory" 1>&2
-	exit 1
+ if [ -n "${BUILD_BASE_DIR}" ]; then
```

Comment:
It seems like here's a spurious space before the `if`, no?

Reply:
Yeah, I wonder if i cut-pasted out of diff output...  can't recall.